serverlessworkflow/modules/ROOT/pages/core/custom-functions-support.adoc

```diff
@@ -537,9 +537,27 @@ You must declare a `functionRef` like the following: (Do not forget to set `asCl
 
 [NOTE]
 ====
-{product_name} generates a CloudEvent ID based on the `source` and the workflow instance ID. In case an ID is set, {product_name} will ignore it and use a generated one.
+{product_name} generates a CloudEvent ID based on the `source` and the workflow instance ID. In case you decide to set an ID, {product_name} will use it and you must ensure it's unique. See below how to set a CloudEvent ID.
```

Comment:
```suggestion
{product_name} generates a CloudEvent ID based on the `source` and the workflow instance ID. In case you decide to set an ID, {product_name} will use it and you must ensure it's unique. Refer to the following example on how to set a CloudEvent ID:
```